src/main/java/org/mockito/internal/invocation/InterceptedInvocation.java

```diff
@@ -119,6 +119,18 @@ public Method getMethod() {
         return (T) arguments[index];
     }
 
+    public MockReference<Object> getMockRef() {
```

Comment:
I don't see these being used anywhere else. Moreover, they are not on the interface `Invocation`. If we were to add them, they have to be on the interface.

Reply:
I'm calling it from mockito-scala, should I add it to the interface then?

Reply:
>should I add it to the interface then?

Please no. MockReference is an internal type and we should not leak it from a public interface.